logs/coexistence-2019-03-13.0.log

```diff
@@ -5754,3 +5754,16 @@ Caused by: org.postgresql.util.PSQLException: This ResultSet is closed.
 	at io.pedestal.interceptor$eval700$fn__701$fn__702.invoke(interceptor.clj:41)
 	at io.pedestal.interceptor.chain$try_f.invokeStatic(chain.clj:54)
 	... 39 common frames omitted
+13:36:46.861 [MLog-Init-Reporter] INFO  com.mchange.v2.log.MLog - MLog clients using slf4j logging.
```

Comment:
delete all log files, then add `logs/*` to .gitignore

Then log file will keep being created, but w/o showing up here

Reply:
will do.
can we continue from here or do you wan t anew commit with comments deleted?

Reply:
fill free to leave them, but the PR comments should make sense on their own